docs/trex_release-notes.md

```diff
@@ -11,6 +11,13 @@ layout: docs
 ----
 See also: [Known Issues]({{site.baseurl}}/docs/trex_known_issues.html)
 
+---
+### Tableau Sandboxed Extensions Development Environment
+*September 2019*
+
+* Sandboxed Extensions are Tableau dashboard extensions that are not permitted to external network requests. Available for testing with Tableau 2019.3. Fully supported in Tableau 2019.4.
```

Comment:
not permitted to *make* external network requests

Reply:
Yikes! This is what @LGraber noticed. I forgot to check for it here.